nixos/modules/services/cluster/k3s/default.nix

```
@@ -432,4 +432,6 @@ in
       };
     };
   };
+
+  meta.maintainers = [ lib.teams.k3s.members ];
```

A list of lists is supported? Should'nt it be like this?

```suggestion
  meta.maintainers = lib.teams.k3s.members;
```

Same below

---

`meta.maintainers` is a list. I also made that mistake just a few minutes ago. :-)

---

Yeah, I need to double check this. It was that way before. But was erroring in nixfmt.

---

Thanks yajo!